server/configmodule.py

```diff
@@ -40,11 +36,11 @@ class AutopushConfig(Config):
 
 
 class SvObsConfig(Config):
```

Comment:
Is this config class still required?

Reply:
This is for svobs.datacommons.org. Can remove this with the flag after migration?

Reply:
yup, sg